src/Command/ImportBoardMemberCommand.php

```diff
@@ -95,9 +95,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this->em->commit();
 
         $output->writeln('Import finish');
-        if (0 < count($this->emailNotFound)) {
+        if ($this->notFoundEmails) {
             $output->writeln('The following email adress are not found in DB');
```

Comment:
typo unrelated to your PR :
~adress~|addresses
~are~|were

Reply:
Done